src/view/screens/Notifications.tsx

```diff
@@ -88,9 +97,48 @@ export const NotificationsScreen = withAuthRequired(
       ),
     )
 
-    const hasNew =
-      store.me.notifications.hasNewLatest &&
-      !store.me.notifications.isRefreshing
+    const ListHeaderComponent = React.useCallback(() => {
+      if (isDesktop) {
+        return (
+          <View
+            style={[
+              pal.view,
+              {
+                flexDirection: 'row',
+                alignItems: 'center',
+                justifyContent: 'space-between',
+                paddingHorizontal: 18,
+                paddingVertical: 12,
+              },
+            ]}>
+            <TextLink
+              type="title-lg"
+              href="/notifications"
+              style={{fontWeight: 'bold'}}
+              text={
+                <>
+                  Notifications{' '}
```

Comment:
This text doesn't change in dark mode. Looks soooo good in light though 🥵 

<img width="675" alt="Screen Shot 2023-09-15 at 2 25 35 PM" src="https://github.com/bluesky-social/social-app/assets/4732330/dba25f24-6fa2-4829-86ed-f0d2f2c41d8f">

Reply:
Good catch, fixed